sw/host/tests/manuf/manuf_cp_unlock_raw/src/main.rs

```diff
@@ -0,0 +1,65 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+//! This test performs a lifecycle transition from `RAW` to `TEST_UNLOCKED0`.
+
+use anyhow::Context;
+use structopt::StructOpt;
+
+use opentitanlib::dif::lc_ctrl::{DifLcCtrlState, DifLcCtrlToken, LcCtrlReg};
+use opentitanlib::io::jtag::{JtagParams, JtagTap};
+use opentitanlib::test_utils::init::InitializeTest;
+use opentitanlib::test_utils::lc_transition;
+
+// Generated by the `lc_raw_unlock_token` Bazel rule from `//rules/lc.bzl`.
+mod lc_raw_unlock_token;
+
+#[derive(Debug, StructOpt)]
+struct Opts {
+    #[structopt(flatten)]
+    init: InitializeTest,
+    #[structopt(flatten)]
+    jtag: JtagParams,
+}
+
+fn main() -> anyhow::Result<()> {
+    let opts = Opts::from_args();
+    opts.init.init_target()?;
+
+    let transport = opts.init.init_target()?;
+
+    // Set the TAP straps for the lifecycle controller and reset.
+    transport
+        .pin_strapping("PINMUX_TAP_LC")?
+        .apply()
+        .context("failed to apply LC TAP strapping")?;
+    transport
+        .reset_target(opts.init.bootstrap.options.reset_delay, true)
+        .context("failed to reset")?;
+
```

Comment:
@jwnrt don't you need to switch to external clk here?

Reply:
The `lc_transition` test util enables the external clock [here](https://github.com/lowRISC/opentitan/blob/f6c35d8ded24c338c3e3a3cfa7dd1d64da3e2e86/sw/host/opentitanlib/src/test_utils/lc_transition.rs#L92).

Does this need to be set earlier? Is there something extra I'm missing that needs to be done to connect the clock?